R/new.R

```diff
@@ -103,6 +103,20 @@ generator_funs$new <- function(...) {
     public_methods  <- merge_vectors(super_struct$public_methods, public_methods)
     private_methods <- merge_vectors(super_struct$private_methods, private_methods)
     active          <- merge_vectors(super_struct$active, active)
+
+    # If `cloneable` property differs between sub and superclass
+    # - super will override sub if super doesn't allow cloning
+    # - sub will override super if super allows cloning
+    if (!identical(cloneable, inherit$cloneable)) {
+      public_methods[["clone"]] <- NULL
+
+      if (!inherit$cloneable) {
+        message(
+          "Superclass ", get_superclassnames(inherit), " has cloneable=FALSE, but subclass ", classname, " has cloneable=TRUE. ",
```

Comment:
`get_superclassnames(inherit)` will return a vector of class names, which will won't print out well. I think that `inherit$classname` will work better.